tests/misc/CachingTest.py

```diff
@@ -130,3 +132,26 @@ def test_caching_results(self):
                 "-f", re.escape(filename),
                 "-b", "LineCountTestBear")
             self.assertIn("This file has", output)
+
+    def test_caching_multi_results(self):
+        """
+        Integration test to assert that results are not dropped when coala is
+        ran multiple times with caching enabled and one section yields a result
+        and second one doesn't.
+        """
+        filename = "tests/misc/test_caching_multi_results/"
+        with bear_test_module():
+            with simulate_console_inputs("0"):
+                retval, output = execute_coala(
+                   coala.main,
+                   "coala",
+                   "-c", filename + ".coafile",
+                   "-f", filename + "test.py")
+                self.assertIn("This file has", output)
+
+            retval, output = execute_coala(
```

Comment:
why doesn't this need simulate console inputs? It should require the same inputs right?